en/components/grids_templates/excel_style_filtering.md

```diff
@@ -309,29 +309,29 @@ If you want to keep the sorting, moving, pinning and hiding features of the colu
 
 @@if (igxName === 'IgxGrid') {
 <div class="sample-container loading" style="height:900px">
-    <iframe id="grid-sample-2-iframe" data-src='{environment:demosBaseUrl}/grid/grid-excel-style-filtering-sample-3' width="100%" height="100%" seamless frameborder="0" class="lazyload"></iframe>
+    <iframe id="grid-sample-3-iframe" data-src='{environment:demosBaseUrl}/grid/grid-excel-style-filtering-sample-3' width="100%" height="100%" seamless frameborder="0" class="lazyload"></iframe>
```

Comment:
Same changes should be applied to the **jp** version of the file as well.

Reply:
Addressed.